assets/css/styles.css

```diff
@@ -12,7 +12,7 @@ margin: 0;
     margin-right: auto;
     /* padding: 20px;
```

Comment:
I suggest you delete unused lines of code to reduce runtime and make your work neater.